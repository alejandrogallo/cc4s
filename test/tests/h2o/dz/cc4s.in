- name: TensorReader
  in:
    fileName: "EigenEnergies.yaml"
  out:
    tensor: EigenEnergies

- name: TensorReader
  in:
    fileName: "CoulombVertex.yaml"
  out:
    tensor: CoulombVertex

- name: DefineHolesAndParticles
  in:
    eigenEnergies: EigenEnergies
  out:
    slicedEigenEnergies: EigenEnergies

- name: SliceOperator
  in:
    slicedEigenEnergies: EigenEnergies
    operator: CoulombVertex
  out:
    slicedOperator: CoulombVertex

- name: VertexCoulombIntegrals
  in:
    slicedCoulombVertex: CoulombVertex
  out:
    coulombIntegrals: CoulombIntegrals

- name: CoupledCluster
  in:
    method: Ccsd
    slicedEigenEnergies: EigenEnergies
    coulombIntegrals: CoulombIntegrals
    slicedCoulombVertex: CoulombVertex
    integralsSliceSize: 100
    maxIterations: 20
    energyConvergence: 1.0E-8
    amplitudesConvergence: 1.0E-8
    mixer:
      type: DiisMixer
      maxResidua: 4
  out:
    energy: CcsdEnergy
    amplitudes: Amplitudes

- name: PerturbativeTriples
  in:
    coulombIntegrals: CoulombIntegrals
    amplitudes: Amplitudes
    slicedEigenEnergies: EigenEnergies
  out:
    {}
