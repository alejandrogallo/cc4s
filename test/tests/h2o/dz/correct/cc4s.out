buildDate: Dec  2 2021 10:45:24
compiler: icc (ICC) 19.1.2.254 20200623
dryRun: 0
hosts:
  0:
    host: n412-034.vsc4.prov
    ranks:
      0: 0
      1: 1
      10: 10
      11: 11
      12: 12
      13: 13
      14: 14
      15: 15
      16: 16
      17: 17
      18: 18
      19: 19
      2: 2
      20: 20
      21: 21
      22: 22
      23: 23
      24: 24
      25: 25
      26: 26
      27: 27
      28: 28
      29: 29
      3: 3
      30: 30
      31: 31
      32: 32
      33: 33
      34: 34
      35: 35
      36: 36
      37: 37
      38: 38
      39: 39
      4: 4
      5: 5
      6: 6
      7: 7
      8: 8
      9: 9
steps:
  0:
    floatingPointOperations: 0
    flops: 0
    in:
      fileName: EigenEnergies.yaml
    name: TensorReader
    out:
      tensor:
        binary: 0
        data: 0x164d238
        dimensions:
          0:
            length: 40
            type: orbital
        energies:
          0: -1.3547964852037999
          1: -0.71624162412494996
          10: 0.28939588405087002
          11: 0.33151845064434998
          12: 0.38572556348800002
          13: 0.40271186975883
          14: 0.43260113444962001
          15: 0.53899960022281002
          16: 0.64747255302890006
          17: 0.65812074173305002
          18: 0.80890038979233003
          19: 0.92980872330205
          2: -0.58564694457710997
          20: 1.1017942792538999
          21: 1.11605810241308
          22: 1.14850059085186
          23: 1.3043985211095701
          24: 1.46524124145144
          25: 1.4814508040575201
          26: 1.58288739637593
          27: 2.0015027381639801
          28: 2.0081227486458402
          29: 2.10768990393349
          3: -0.50921622549824996
          30: 2.3620280549229
          31: 2.4753343053626899
          32: 2.6200056895272099
          33: 2.7127165018631598
          34: 2.9689662522920899
          35: 3.67307420504949
          36: 3.68992496028488
          37: 3.70196976168786
          38: 4.0195384608236999
          39: 4.3012603606542701
          4: 0.035332851045540001
          5: 0.057942254975680001
          6: 0.17402402808674
          7: 0.19692534033067
          8: 0.22240503898927999
          9: 0.23169022231684
        fermiEnergy: 0
        indices:
          orbital:
            type: spatial
        scalarType: real64
        unit: 1
        version: v1.0
    realtime: 0.010755440
  1:
    floatingPointOperations: 0
    flops: 0
    in:
      fileName: CoulombVertex.yaml
    name: TensorReader
    out:
      tensor:
        binary: 1
        data: 0x1656c68
        dimensions:
          0:
            length: 1099
            type: momentum
          1:
            length: 40
            type: orbital
          2:
            length: 40
            type: orbital
        indices:
          momentum:
            type: halfGrid
          orbital:
            type: spatial
        scalarType: complex64
        unit: 1
        version: v1.0
    realtime: 0.045400572
  2:
    floatingPointOperations: 0
    flops: 0
    in:
      eigenEnergies:
        binary: 0
        data: 0x164d238
        dimensions:
          0:
            length: 40
            type: orbital
        energies:
          0: -1.3547964852037999
          1: -0.71624162412494996
          10: 0.28939588405087002
          11: 0.33151845064434998
          12: 0.38572556348800002
          13: 0.40271186975883
          14: 0.43260113444962001
          15: 0.53899960022281002
          16: 0.64747255302890006
          17: 0.65812074173305002
          18: 0.80890038979233003
          19: 0.92980872330205
          2: -0.58564694457710997
          20: 1.1017942792538999
          21: 1.11605810241308
          22: 1.14850059085186
          23: 1.3043985211095701
          24: 1.46524124145144
          25: 1.4814508040575201
          26: 1.58288739637593
          27: 2.0015027381639801
          28: 2.0081227486458402
          29: 2.10768990393349
          3: -0.50921622549824996
          30: 2.3620280549229
          31: 2.4753343053626899
          32: 2.6200056895272099
          33: 2.7127165018631598
          34: 2.9689662522920899
          35: 3.67307420504949
          36: 3.68992496028488
          37: 3.70196976168786
          38: 4.0195384608236999
          39: 4.3012603606542701
          4: 0.035332851045540001
          5: 0.057942254975680001
          6: 0.17402402808674
          7: 0.19692534033067
          8: 0.22240503898927999
          9: 0.23169022231684
        fermiEnergy: 0
        indices:
          orbital:
            type: spatial
        scalarType: real64
        unit: 1
        version: v1.0
    name: DefineHolesAndParticles
    out:
      slicedEigenEnergies:
        dimensions:
          0:
            length: 40
            type: orbital
        holesCount: 4
        indices:
          orbital:
            type: spatial
        particlesCount: 36
        scalarType: real64
        slices:
          h: 0x164fb30
          p: 0x164db70
        unit: 1
    realtime: 0.000832881
  3:
    floatingPointOperations: 0
    flops: 0
    in:
      operator:
        binary: 1
        data: 0x1656c68
        dimensions:
          0:
            length: 1099
            type: momentum
          1:
            length: 40
            type: orbital
          2:
            length: 40
            type: orbital
        indices:
          momentum:
            type: halfGrid
          orbital:
            type: spatial
        scalarType: complex64
        unit: 1
        version: v1.0
      slicedEigenEnergies:
        dimensions:
          0:
            length: 40
            type: orbital
        holesCount: 4
        indices:
          orbital:
            type: spatial
        particlesCount: 36
        scalarType: real64
        slices:
          h: 0x164fb30
          p: 0x164db70
        unit: 1
    name: SliceOperator
    out:
      slicedOperator:
        binary: 1
        dimensions:
          0:
            length: 1099
            type: momentum
          1:
            length: 40
            type: orbital
          2:
            length: 40
            type: orbital
        indices:
          momentum:
            type: halfGrid
          orbital:
            type: spatial
        scalarType: complex64
        slices:
          hh: 0x1650e60
          hp: 0x164c800
          ph: 0x1653f80
          pp: 0x16463a0
        unit: 1
        version: v1.0
    realtime: 0.000798066
  4:
    floatingPointOperations: 0
    flops: 0
    in:
      slicedCoulombVertex:
        binary: 1
        dimensions:
          0:
            length: 1099
            type: momentum
          1:
            length: 40
            type: orbital
          2:
            length: 40
            type: orbital
        indices:
          momentum:
            type: halfGrid
          orbital:
            type: spatial
        scalarType: complex64
        slices:
          hh: 0x1650e60
          hp: 0x164c800
          ph: 0x1653f80
          pp: 0x16463a0
        unit: 1
        version: v1.0
    name: VertexCoulombIntegrals
    out:
      coulombIntegrals:
        dimensions:
          0:
            length: 40
            type: orbital
          1:
            length: 40
            type: orbital
          2:
            length: 40
            type: orbital
          3:
            length: 40
            type: orbital
        indices:
          orbital:
            type: spatial
        scalarType: real64
        slices:
          hhhh: 0x165bfc0
          hhhp: 0x16612f0
          hhph: 0x165e950
          hhpp: 0x1663e00
          hphh: 0x165d3f0
          hphp: 0x1662830
          hpph: 0x165fe60
          hppp: 0x16652a0
          phhh: 0x165ca60
          phhp: 0x1661dd0
          phph: 0x165f330
          phpp: 0x1664800
          pphh: 0x165de80
          pphp: 0x16632b0
          ppph: 0x1660910
          pppp: 0x1665d40
        unit: 1
    realtime: 0.003376161
  5:
    floatingPointOperations: 119516258698
    flops: 20522506210.843185
    in:
      amplitudesConvergence: 1e-08
      coulombIntegrals:
        dimensions:
          0:
            length: 40
            type: orbital
          1:
            length: 40
            type: orbital
          2:
            length: 40
            type: orbital
          3:
            length: 40
            type: orbital
        indices:
          orbital:
            type: spatial
        scalarType: real64
        slices:
          hhhh: 0x165bfc0
          hhhp: 0x16612f0
          hhph: 0x165e950
          hhpp: 0x1663e00
          hphh: 0x165d3f0
          hphp: 0x1662830
          hpph: 0x165fe60
          hppp: 0x16652a0
          phhh: 0x165ca60
          phhp: 0x1661dd0
          phph: 0x165f330
          phpp: 0x1664800
          pphh: 0x165de80
          pphp: 0x16632b0
          ppph: 0x1660910
          pppp: 0x1665d40
        unit: 1
      energyConvergence: 1e-08
      integralsSliceSize: 100
      levelShift: 0
      maxIterations: 20
      method:
        2cc: 0
        dcsd: 0
        integralsSliceSize: 4
        ppl: 1
        type: Ccsd
      mixer:
        maxResidua: 4
        type: DiisMixer
      slicedCoulombVertex:
        binary: 1
        dimensions:
          0:
            length: 1099
            type: momentum
          1:
            length: 40
            type: orbital
          2:
            length: 40
            type: orbital
        indices:
          momentum:
            type: halfGrid
          orbital:
            type: spatial
        scalarType: complex64
        slices:
          hh: 0x1650e60
          hp: 0x164c800
          ph: 0x1653f80
          pp: 0x16463a0
        unit: 1
        version: v1.0
      slicedEigenEnergies:
        dimensions:
          0:
            length: 40
            type: orbital
        holesCount: 4
        indices:
          orbital:
            type: spatial
        particlesCount: 36
        scalarType: real64
        slices:
          h: 0x164fb30
          p: 0x164db70
        unit: 1
    name: CoupledCluster
    out:
      amplitudes: 0x18eb578
      convergenceReached: 1
      energy:
        direct: -0.35610283424411437
        exchange: 0.12862870324596989
        secondOrder: -0.21973005804253531
        unit: 1
        value: -0.22747413099814448
    realtime: 5.823667805
  6:
    floatingPointOperations: 851677056
    flops: 7852231207.4827614
    in:
      amplitudes: 0x18eb578
      coulombIntegrals:
        dimensions:
          0:
            length: 40
            type: orbital
          1:
            length: 40
            type: orbital
          2:
            length: 40
            type: orbital
          3:
            length: 40
            type: orbital
        indices:
          orbital:
            type: spatial
        scalarType: real64
        slices:
          hhhh: 0x165bfc0
          hhhp: 0x16612f0
          hhph: 0x165e950
          hhpp: 0x1663e00
          hphh: 0x165d3f0
          hphp: 0x1662830
          hpph: 0x165fe60
          hppp: 0x16652a0
          phhh: 0x165ca60
          phhp: 0x1661dd0
          phph: 0x165f330
          phpp: 0x1664800
          pphh: 0x165de80
          pphp: 0x16632b0
          ppph: 0x1660910
          pppp: 0x1665d40
        unit: 1
      slicedEigenEnergies:
        dimensions:
          0:
            length: 40
            type: orbital
        holesCount: 4
        indices:
          orbital:
            type: spatial
        particlesCount: 36
        scalarType: real64
        slices:
          h: 0x164fb30
          p: 0x164db70
        unit: 1
    name: PerturbativeTriples
    out:
      energy:
        triples: -0.0052396080185706413
    realtime: 0.108463064
version: heads/develop-0-g3297689